aiohttp/worker.py

```diff
@@ -34,7 +34,9 @@ def run(self):
         self._runner = ensure_future(self._run(), loop=self.loop)
 
         try:
-            self.loop.run_until_complete(self._runner)
+            self.loop.run_until_complete(asyncio.gather(self._runner,
```

Comment:
I'm not sure I get you right. Do you mean to just schedule `self.wsgi.startup()` with `ensure_future()` and keep the rest as it was done before?

```
ensure_future(self.wsgi.startup(), loop=self.loop)
self._runner = ensure_future(self._run(), loop=self.loop)

try:
    self.loop.run_until_complete(self._runner)
finally:
    self.loop.close()
```